djangopg/test_settings.py

```diff
@@ -1,15 +1,30 @@
 # -*- coding: utf-8 -*-
```

Comment:
This should not be in the package. Rather figure out a way where we keep test related files apart from djangopg package.

Reply:
When I change to setup.cfg to:
```
[tool:pytest]
DJANGO_SETTINGS_MODULE = test_settings
```
I got: 
```python
Traceback (most recent call last):
File "manage.py", line 10, in <module>
  execute_from_command_line(sys.argv)
File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
  utility.execute()
File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 303, in execute
  settings.INSTALLED_APPS
File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 48, in __getattr__
  self._setup(name)
File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 44, in _setup
  self._wrapped = Settings(settings_module)
File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 92, in __init__
  mod = importlib.import_module(self.SETTINGS_MODULE)
File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
  __import__(name)
ImportError: No module named test_settings
```
Will try some different things...